src/Video.js

```diff
@@ -577,16 +578,18 @@ export default class Video extends React.Component {
   getVideoDimensions = () => {
     let width, height;
     let {
-      props: { maxWidth, forcedWidth },
+      props: { maxWidth, live },
       state: { fullscreen }
     } = this;
-    if (!!forcedWidth) {
-      windowWidth = forcedWidth;
-    }
 
     if (this.props.youtubeId) {
       if (fullscreen && !isTablet) {
-        return { width: "100%", height: "100%" };
+        return {
+          width: "100%",
+          aspectRatio: 16/9,
+          height: live ? undefined : "100%",
+          backgroundColor: 'green'
```

Comment:
Is this needed?

Reply:
Oh, definitely not, that was for testing only.. I dont think its visible, but will remove it.